balena/utils.py

```diff
@@ -0,0 +1,15 @@
+import numbers
+
+
+def is_id(value):
+    """
+    Return True, if the input value is a valid ID. False otherwise.
```

Comment:
this function just checks if the input could be an integer, not necessarily that it's a valid id, right?